### ISSUE TITLE
date received for an application is 18 years ago!

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/yorke_peninsula_council_sa_development_applications#4]
See for example https://www.planningalerts.org.au/applications/1187387

"We found this application for you on the planning authority's website 15 days ago. It was received by them about **18 years earlier**."

Is this related to #3 in some way?

**Comment by MichaelBone** [comment]
yes, I think this is related to #3.  It's happened because the scraper is gradually, randomly scraping historic applications in a (misguided?) attempt to build up a complete picture of all applications over time (ie. without obtaining them all at once and possibly overloading the Yorke Peninsula web site).
